sale_elaboration/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+#. Go to *Sale > Quotations > Create*.
+   Create a sales order and select an elaboration in the line.
+#. Go to the picking created by this sale order and validate it.
```

Comment:
I was testing this in runbot. In this step, how do the warehouse guy know what elaboration is needed. No info is supplied in the picking.

Reply:
And once I confirm the picking, another picking is created to compensate the stock of the elaboration product. Anyway, this shouldn't be the case if the product is a consumable, isn't it?

Reply:
Elaboration products are services.